__tests__/app.test.js

```diff
@@ -60,3 +60,40 @@ describe("/api", ()=>{
         });
     });
 });
+
+describe("/api/articles/:article_id", () => {
+    test("GET: 200 - responds with the correct article object for a valid id", () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: 1,
+                    body: expect.any(String),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                });
+            });
+    });
+    test("GET: 404 - responds with an error message for an id that does not exist", ()=>{
+        return request(app)
+        .get("/api/articles/99999")
```

Comment:
What does 99999 mean, you could store it in a variable `const NON_EXISTANT_ID = 999999` that would give it much more semantic meaning, this is something to consider when using any seemingly random numbers